### ISSUE TITLE
Missing pymysql3

### ISSUE THREAD

**Issue by AfzalH** [EasyEngine/easyengine#969]
Can't install ee with the command `wget -qO ee rt.cx/ee && sudo bash ee`. It was fine even 20 hours ago.
Here's the generated error near the end of the installation

```
Searching for pymysql3==0.4
Reading https://pypi.python.org/simple/pymysql3/
Couldn't find index page for 'pymysql3' (maybe misspelled?)
```

It seems that the pymysql3 page/package is not there anymore.
The first page returned by google search for pymsql3 is also showing a not found page
https://pypi.python.org/pypi/PyMySQL3/

Is it possible to give it a quick fix or any workaround?

**Comment by jatiyanon** [comment]
same here

**Comment by tiagoandre** [comment]
same here

**Comment by mtstedman** [comment]
Ditto on this issue.  Can't install EE on fresh Ubuntu 16.04 installs as of yesterday.  Same error.

**Comment by jatiyanon** [comment]
and now have an error in fresh install.

Setting up database             Oops Something went wrong !!
Calling cleanup actions ...
'ee_mysql_grant_host'

**Comment by farukipk** [comment]
I have the same problem. I can not add the site. Is there a solution?

**Comment by AfzalH** [comment]
It seems pymysql isn't a drop in replacement for pymysql3. I also tried changing pymysql3 to pymysql by creating a fork. It removed the installation error but site management isn't working

**Comment by farukipk** [comment]
Have you made a mistake while adding the website?
`ee_mysql_grant_host`

**Comment by AfzalH** [comment]
I'm not trying with easyengine anymore. Trying to do the site management myself. So far I've come up with this https://github.com/afzalh/lara-server
It needs manual commands but everything is under users control so no dependency (except the standard packages)

**Comment by mbtamuli** [comment]
We're looking into this.

**Comment by VirtuBox** [comment]
Hello guys, the only workaround I have found is to copy the folder /usr/local/lib/python3.5/dist-packages/ from another server with pymysql3 at the moment.
I have uploaded this folder in a zip file on  [Github](https://github.com/VirtuBox/easyengine/raw/master/dist.zip) if you want to try this temporary solution.
 Folder content is from a server running ubuntu 16.04 LTS with easyengine.

**Comment by princev558** [comment]
thanks VirtuBox for solution. now its working

**Comment by farukipk** [comment]
Which code did you download?
I'm not an expert on this. Could you share the code completely, please.

**Comment by princev558** [comment]
1. cd /usr/local/lib/python3.5/
2. rm -r dist-packages
3. sudo apt-get install unzip
4. wget https://github.com/VirtuBox/easyengine/raw/master/dist.zip
5. unzip dist.zip
6. wget -qO ee rt.cx/ee && sudo bash ee # install easyengine
7. sudo ee site create example.com --wp # install wordpress on example.com
and now try to install easy engine

**Comment by VirtuBox** [comment]
To avoid errors with permissions and files owners : 
```
sudo wget https://github.com/VirtuBox/easyengine/raw/master/dist.zip -O  /usr/local/lib/python3.5/dist.zip

sudo apt install unzip
sudo rm -rf dist-packages
sudo unzip dist.zip
sudo rm dist.zip
sudo chown root:staff dist-packages/*.egg
```
Then you should be able to install easyengine. If you have already installed EE, it doesn't matter, you can simply use it and create your first site.

**Comment by farukipk** [comment]
Worked. I love you. Thank you so much.

**Comment by mbtamuli** [comment]
This has been fixed in [EasyEngine v3.7.5](https://github.com/EasyEngine/easyengine/releases/tag/v3.7.5). The [Travis build](https://travis-ci.org/EasyEngine/easyengine/builds/360103042) passed.

Thanks @VirtuBox for providing a workaround for users till this was fixed.

Closing this issue.

**Comment by janiosarmento** [comment]
Shouldn't the new version be 3.7.6 or at least 3.7.5.1?

**Comment by fagnerguima** [comment]
The version with the problem already is 3.7.5. I think that the fixed version must be 3.7.6. Isn't?
